spec/spec_helper.rb

```diff
@@ -30,8 +30,16 @@
   SimpleCov.root(File.expand_path('../..', __FILE__))
   SimpleCov.formatter = Coveralls::SimpleCov::Formatter
   SimpleCov.start('rails') do
+    add_filter '/.internal_test_app'
+    add_filter '/bin'
+    add_filter '/db'
+    add_filter '/config'
```

Comment:
I'm pretty sure /config is already filtered by the 'rails' profile See https://github.com/colszowka/simplecov#profiles